threedi_scenario_downloader/downloader.py

```diff
@@ -217,8 +215,17 @@ def create_raster_task(
         pixelsize_x = resolution
         pixelsize_y = resolution
 
-    width = math.ceil((x2 - x1) / pixelsize_x)
-    height = math.ceil((y2 - y1) / pixelsize_y)
+    width = abs((x2 - x1) / pixelsize_x)
+    height = abs((y2 - y1) / pixelsize_y)
+
+    if not width.is_integer():
+        width = math.ceil(width)
+        x2 = (width * pixelsize_x) + x1
+    if not height.is_integer():
+        height = math.ceil(height)
+        y2 = (height * pixelsize_y) + y1
```

Comment:
Ik denk dat dit fout gaat. Als `width` WEL een integer is wordt `x2` niet gezet. Dus misschien die `math.ceil()` gewoon altijd gebruiken?

```suggestion
    # Optie 1: x2-regel naar links
    if not width.is_integer():
        width = math.ceil(width)
    x2 = (width * pixelsize_x) + x1
    # Optie 2: gewoon altijd .ceil aanroepen.
    height = math.ceil(height)
    y2 = (height * pixelsize_y) + y1
```

Reply:
Ok!
De commentaarregel die je hebt toegevoegd verduidelijkt het ook prima voor iemand anders die ernaar gaat kijken.